style.css

```diff
@@ -382,3 +374,223 @@ form textarea {
   display: flex;
   justify-content: center;
 }
+
+.btn-transition:hover {
+  transition: width 2s, height 2s, background-color 2s, transform 2s;
+  background: #6070ff;
+  color: white;
+  width: 30%;
+}
+
+nav {
+  display: none;
+}
+
+@media only screen and (min-width: 992px) {
+  .toggleNav {
+    display: none;
+  }
+
+  nav {
+    display: flex;
+  }
+
+  nav ul {
+    display: flex;
+    list-style-type: none;
+  }
+
+  nav ul li {
+    margin: 10px;
+    padding: 10px;
+    font-family: Poppins, sans-serif;
+    margin-right: 70px;
+    font-size: 120%;
+  }
+
+  nav ul li a {
+    color: #344563;
+  }
+
+  .br {
+    display: none;
+  }
+
+  .container {
+    margin-left: 13vw;
+    margin-right: 32vw;
+  }
+
+  .h1 {
+    line-height: 60px;
+    letter-spacing: 0.38px;
+  }
+
+  .icons {
+    display: grid;
+    grid-template-columns: repeat(5, 20%);
+    list-style: none;
+    text-decoration: none;
+    padding-top: 20px;
+    align-items: flex-start;
+    width: 180px;
+  }
+
+  .connect {
+    font-size: 20px;
+  }
+
+  .card-w1 {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    height: 496px;
+    width: 1156px;
+    margin-top: 142px;
+    margin-bottom: 140px;
+    margin-left: 8vw;
```

Comment:
- [ ] **OPTIONAL**
Kindly take another look at the spacings between the projects' cards (margin and padding) to match the given design.
I'd be great if you worked on this.
### **Your design**
![Screenshot1](https://user-images.githubusercontent.com/47625407/121763274-f7bf8e80-cb3a-11eb-94e5-9f9cdf121389.png)
### **[Original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)**
![Screenshot2](https://user-images.githubusercontent.com/47625407/121763283-05751400-cb3b-11eb-8278-1f25833e9b01.png)

- [ ] **OPTIONAL**
Please consider the required changes asked earlier about the images of the projects' cards. The image should take all the space as the given design.
### **Your design**
![Screenshot3](https://user-images.githubusercontent.com/47625407/121763326-57b63500-cb3b-11eb-8dc6-00207902b325.png)
### **Original design**
![Screenshot4](https://user-images.githubusercontent.com/47625407/121763329-6270ca00-cb3b-11eb-9bab-af96be120502.png)

> Kindly keep in mind that these are mandatory changes according to the project's requirements and I'm sorry that they didn't show up on the first review.